index.js

```diff
@@ -82,6 +128,30 @@ module.exports = {
     return tree;
   },
 
+  postBuild({ directory }) {
+    if (!process.env.EMBER_CLI_ELECTRON) {
+      return tree;
+    }
+
+    //
+    // electron-forge needs our package.json to have a main entry pointing to
+    // electron.js. I can't figure out how to get broccoli to edit JSON for us,
+    // so we just do it manually once the build is complete.
+    //
```

Comment:
@bendemboski Just curious, under what circumstances would an ember-electron project not already have package.json's `main` property set to `ember-electron/electron.js`?